src/tests/ftest/pool/pool_svc.py

```diff
@@ -130,6 +129,8 @@ def test_pool_svc(self):
                         "DaosServerManager.stop_ranks([{}])".format(
                             pool_leader))
 
+                self.pool.wait_for_rebuild(False, interval=1)
```

Comment:
Minor: I believe you don't need to pass interval=1. The default is 1.

Reply:
Actually I'd like to change it to:
    self.pool.wait_for_rebuild(to_start=False, interval=1) for the sake of readability.